parachain/runtime/testnet-kintsugi/src/lib.rs

```diff
@@ -1219,7 +1202,6 @@ construct_runtime! {
         Democracy: democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 70,
         TechnicalCommittee: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 71,
         TechnicalMembership: pallet_membership::{Pallet, Call, Storage, Event<T>, Config<T>} = 72,
-        Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 73,
```

Comment:
Are we sure that we want to remove the treasury account? No objection but just want to know if we are absolutely sure. If we do can you leave a comment on the runtimes with the index that the treasury pallet was at? Same as I did when we removed the relay pallet before. Also we should probably clear up treasury storage at the same time we do this and let OpenSquare know because this will break their interface.

Reply:
We're not removing any account? The account is still defined here 
https://github.com/interlay/interbtc/blob/62256a63f836287029bc9cebe5340f34596b0d52/parachain/runtime/kintsugi/src/lib.rs#L630-L631

Regarding clearing storage, maybe it's easier to just batch a `killPrefix` in the proposal?